packages/tab-bar/README.md

```diff
@@ -1,64 +1,278 @@
+```html
+<mwc-tab-bar>
+  <mwc-tab
+      label="tab one"
+      icon="accessibility"
+      stacked
+      isMinWidthIndicator>
+  </mwc-tab>
+  <mwc-tab
+      label="tab two"
+      icon="exit_to_app"
+      stacked
+      isMinWidthIndicator>
+  </mwc-tab>
+  <mwc-tab
+      label="tab three"
+      icon="camera"
+      stacked
+      isMinWidthIndicator>
+  </mwc-tab>
+</mwc-tab-bar>
+```
 
-  Tools like [WebPack](https://webpack.js.org/) and [Rollup](https://rollupjs.org/) can also be used to serve and/or bundle mwc-tab-bar.
+### Icon Indicator (w/ stacked icon)
 
-## Supported Browsers
+<img src="images/icon-indicator.png" width="600px">
 
-The last 2 versions of all modern browsers are supported, including
-Chrome, Safari, Opera, Firefox, Edge. In addition, Internet Explorer 11 is also supported.
\ No newline at end of file
+```html
+<mwc-tab-bar>
+  <mwc-tab
+      icon="camera"
+      indicatorIcon="donut_large">
+  </mwc-tab>
+  <mwc-tab
+      icon="accessibility"
+      indicatorIcon="donut_large">
+  </mwc-tab>
+  <mwc-tab
+      icon="exit_to_app"
+      indicatorIcon="donut_large">
+  </mwc-tab>
+</mwc-tab-bar>
+```
+
+### Scrollable
+
+When the contents of `mwc-tab-bar` exceed its width, the overflow tabs are made
+scrollable along the x axis.
+
+<img src="images/scrollable.png" width="600px">
+
+```html
+<style>
+  mwc-tab-bar {
+    border: solid black 1px;
+  }
+</style>
+<mwc-tab-bar>
+  <mwc-tab label="Tab one"></mwc-tab>
+  <mwc-tab label="Tab two"></mwc-tab>
+  <mwc-tab label="Tab three"></mwc-tab>
+  <mwc-tab label="Tab four"></mwc-tab>
+  <mwc-tab label="Tab five"></mwc-tab>
+  <mwc-tab label="Tab six"></mwc-tab>
+  <mwc-tab label="Tab seven"></mwc-tab>
+  <mwc-tab label="Tab eight"></mwc-tab>
+  <mwc-tab label="Tab nine"></mwc-tab>
+  <mwc-tab label="Tab ten"></mwc-tab>
+  <mwc-tab label="Tab eleven"></mwc-tab>
+  <mwc-tab label="Tab twelve"></mwc-tab>
+</mwc-tab-bar>
+```
+
+### Styled
+
+_Note: example is in the state of hovering over the first tab._
+
+<img src="images/styled.png" width="600px">
+
+```html
+<style>
+  mwc-tab-bar {
+    --mdc-theme-primary: red;
+    --mdc-text-transform: none;
+    --mdc-tab-border-radius: 20px 20px 0px 0px;
+    --mdc-tab-color-default: green;
+    --mdc-tab-text-label-color-default: green;
+    --mdc-tab-stacked-height: 100px;
+  }
+</style>
+<mwc-tab-bar>
+  <mwc-tab
+      label="tab one"
+      icon="accessibility"
+      stacked
+      isMinWidthIndicator>
+  </mwc-tab>
+  <mwc-tab
+      label="tab two"
+      icon="exit_to_app"
+      stacked
+      isMinWidthIndicator>
+  </mwc-tab>
+  <mwc-tab
+      label="tab three"
+      icon="camera"
+      stacked
+      isMinWidthIndicator>
+  </mwc-tab>
+</mwc-tab-bar>
+```
+
+## API
+
+### Slots
+
+| Name              |	Description
+| ----------------- | -------------
+| _default_         |	[`mwc-tab`](https://github.com/material-components/material-components-web-components/tree/master/packages/tab) elements to display.
```

Comment:
s/default/icon

Reply:
ping (new PR?)

Reply:
oh no, that's right, you're thinking of `packages/tab/README.md` not `tab-bar`

Reply:
sorry, I thought this comment was on the old PR and thought you were commenting on `tab`